eqcorrscan/utils/plotting.py

```diff
@@ -852,43 +853,42 @@ def detection_multiplot(stream, template, times, streamcolour='k',
                        for j in range(len(image.data))]
         axis.plot(image_times, image.data / max(image.data),
                   streamcolour, linewidth=1.2)
-        for k, time in enumerate(times):
-            lagged_time = UTCDateTime(time) + (template_tr.stats.starttime -
-                                               mintime)
-            lagged_time = lagged_time.datetime
-            template_times = [lagged_time +
-                              dt.timedelta((j * template_tr.stats.delta) /
-                                           86400)
-                              for j in range(len(template_tr.data))]
-            # Normalize the template according to the data detected in
-            try:
-                normalizer = max(image.data[int((template_times[0] -
-                                                image_times[0]).
-                                                total_seconds() /
-                                                image.stats.delta):
-                                            int((template_times[-1] -
-                                                 image_times[0]).
-                                                total_seconds() /
-                                                image.stats.delta)] /
-                                 max(image.data))
-            except ValueError:
-                # Occurs when there is no data in the image at this time...
-                normalizer = max(image.data)
-            normalizer /= max(template_tr.data)
-            axis.plot(template_times,
-                      template_tr.data * normalizer,
-                      templatecolour, linewidth=1.2)
+        time = times[i]
+        time_dt = time.datetime
+        template_times = [time_dt +
+                          dt.timedelta((j * template_tr.stats.delta) /
+                                       86400)
+                          for j in range(len(template_tr.data))]
+        # Normalize the template according to the data detected in
+        normalizer = max(image.data[int((template_times[0] -
+                                        image_times[0]).
+                                        total_seconds() /
+                                        image.stats.delta):
+                                    int((template_times[-1] -
+                                         image_times[0]).
+                                        total_seconds() /
+                                        image.stats.delta)] /
+                         max(image.data))
+        normalizer /= max(template_tr.data)
```

Comment:
You removed the try/except statement, any reason for this? I think it looks (from my comment) like it was necessary when there was no data present?